metadata.rb

```diff
@@ -1,8 +1,31 @@
 name             "rubygems"
-maintainer       "Apache v2"
-maintainer_email "Sean OMeara <someara@opscode.com>"
-license          "Apache2"
-description      "Installs/Configures rubygems"
-long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          "0.3.0"
+maintainer       "Ryan Hass"
+maintainer_email "Ryan Hass <rhass+community-cookbooks@chef.io>"
+license          "apachev2"
+description      "Configures rubygems and bundler"
+long_description IO.read(File.join(File.dirname(__FILE__), "README.md"))
+version          "0.4.0"
 
+supports 'amazon'
+supports 'centos'
+supports 'debian'
+supports 'fedora'
+supports 'freebsd'
+supports 'mac_os_x', '>= 10.7.0'
+supports 'mac_os_x_server', '>= 10.7.0'
+supports 'opensuse'
+supports 'opensuseleap'
+supports 'oracle'
+supports 'redhat'
+supports 'scientific'
+supports 'smartos'
+supports 'solaris'
+supports 'suse'
+supports 'ubuntu'
+supports 'windows'
+supports 'zlinux'
+
+issues_url "https://github.com/chef-cookbooks/rubygems/issues" if respond_to?(:issues_url)
+source_url "https://github.com/chef-cookbooks/rubygems" if respond_to?(:source_url)
+
+chef_version ">= 12.8.1"
```

Comment:
You should probably pull in compat_resource then to get the backported code all the way to 12.1